src/meshapi/util/uisp_import/update_objects.py

```diff
@@ -75,13 +75,20 @@ def update_device_from_uisp_data(
 
 def update_link_from_uisp_data(
     existing_link: Link,
+    uisp_link_id: str,
     uisp_from_device: Device,
     uisp_to_device: Device,
     uisp_status: Link.LinkStatus,
     uisp_session: Optional[requests.Session] = None,
 ) -> List[str]:
     change_messages = []
 
+    if uisp_link_id != existing_link.uisp_id:
+        existing_link.uisp_id = uisp_link_id
+        # TODO: Convert this to a log warning instead of an admin message if this fix holds
+        #  GH issue: https://github.com/nycmeshnet/meshdb/issues/691
```

Comment:
I will likely address this TODO today, after we see how this change works in dev